src/components/Morpher.js

```diff
@@ -149,6 +149,7 @@ const Morpher = () => {
       clearInterval(morphInterval)
       clearTimeout(morphTimeout)
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Yeah I think its ok for this case. I've seen this before too, I wasn't sure of doing it haha